src/selectallui.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module select-all/selectallui
+ */
+
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+import ButtonView from '@ckeditor/ckeditor5-ui/src/button/buttonview';
+
+import selectAllIcon from '../theme/icons/select-all.svg';
+
+/**
+ * The select all UI feature.
+ *
+ * It registers the `'selectAll'` UI button in the editor's
+ * {@link module:ui/componentfactory~ComponentFactory component factory}. When clicked, the button
+ * executes the {@link module:select-all/selectallcommand~SelectAllCommand select all command}.
+ *
+ * @extends module:core/plugin~Plugin
+ */
+export default class SelectAllUI extends Plugin {
+	/**
+	 * @inheritDoc
+	 */
+	static get pluginName() {
+		return 'SelectAllUI';
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	init() {
+		const editor = this.editor;
+
+		editor.ui.componentFactory.add( 'selectAll', locale => {
+			const command = editor.commands.get( 'selectAll' );
+			const view = new ButtonView( locale );
+			const t = locale.t;
+
+			view.set( {
+				label: t( 'Select all' ),
+				icon: selectAllIcon,
+				keystroke: 'CTRL+A',
```

Comment:
```suggestion
				keystroke: 'Ctrl+A',
```

We're not consistent here (it's wrong in basic-styles only) but that was the consensus to write Ctrl/Shift/Alt.

Reply:
That's also what you used later on in the manual test.